test/tests/ctypes_test.py

```diff
@@ -0,0 +1,8 @@
+# skip-if: True
```

Comment:
I guess my theory was that that other bug might not affect this small test -- for instance there's no multithreading, and it's a short enough test that a GC won't happen.  Would you mind humoring me and trying enabling it? :)  I think it's unlikely that it will fail, but if it does maybe it will give some good information on what's going on.